zebROS_ws/src/behaviors/src/2023_updown_elevater_server.cpp

```diff
@@ -56,20 +63,25 @@ class ElevaterAction2023 {
 	double elev_cur_position_;
   double position_offset_ = 0;
   double position_tolerance_ = 0.02;
+  double safety_high_position_ = -999;
+  double safety_low_position_ = -999;
+  size_t elevater_master_idx; 
 
 public:
 
   ElevaterAction2023(std::string name) :
     as_(nh_, name, boost::bind(&ElevaterAction2023::executeCB, this, _1), false),
+    ac_fourber_("/elevator/fourber_server_2023" , true), 
     nh_params_(nh_, "elevater_server_2023"),
     action_name_(name),
-    ddr_(nh_params_)
+    ddr_(nh_params_) 
   {
     // time to load all 16 params!
     // the idea here is that the customization is good, and if values are the same than thats great
     // current mech designs do have us going to diffrent heights based on what we have, and this is essentially the lookup so that every node doesn't have to do this
     // we will never (hopefully) need more posibilites than this, so we are prepared for everything mech could come up with
     // also if we for certain pieces/heights we can change height if we notice something off
+    elevater_master_idx = std::numeric_limits<size_t>::max();
```

Comment:
Code like this in the constructor can be moved to just initialize the var where it is declared instead - puts all of the simple initialization code in one place